aws/assume-role/README.md

```diff
@@ -206,15 +271,15 @@ tasks:
 
   - key: assume-role
     use: aws-cli
-    call: aws/assume-role 2.0.0 # replaces aws/assume-role 1.1.4
+    call: aws/assume-role 2.0.0
     with:
       oidc-token: ${{ vaults.your-vault.oidc.your-token }}
       region: us-east-2
       role-to-assume: arn:aws:iam::your-account-id:role/your-role
 
   - key: chain-role
-    use: aws-cli
-    call: aws/assume-role 2.0.0 # replaces aws/assume-role 1.1.4
+    use: assume-role
```

Comment:
Ope it changed back in this one 🤔 

This also contradicts an earlier example in the readme in this PR

Reply:
https://github.com/rwx-research/mint-leaves/pull/190#discussion_r1946916536